packages/blocks/shuffle/src/components/item.tsx

```diff
@@ -0,0 +1,123 @@
+import React, { FunctionComponent, useState } from "react";
+import AddIcon from "@mui/icons-material/Add";
+import CloseIcon from "@mui/icons-material/Close";
+import DragIndicatorIcon from "@mui/icons-material/DragIndicator";
+import { Box, IconButton, ListItem, TextField, Paper } from "@mui/material";
+import { useSortable } from "@dnd-kit/sortable";
+import { CSS } from "@dnd-kit/utilities";
+
+type ItemProps = {
+  id: string;
+  value: string;
+  isDragging?: boolean;
+  onValueChange?: (value: string) => void;
+  onItemBlur?: () => void;
+  onAdd?: () => void;
+  onDelete?: () => void;
+};
+
+export const Item: FunctionComponent<ItemProps> = ({
+  id,
+  value,
+  isDragging,
+  onValueChange,
+  onItemBlur,
+  onAdd,
+  onDelete,
+}) => {
+  const { attributes, listeners, setNodeRef, transform, transition } =
+    useSortable({ id });
+
+  const style = {
+    transform: CSS.Transform.toString(transform),
+    transition,
```

Comment:
Is this not supposed to have `CSS.Transition.toString` applied to it?

Reply:
`transition` here comes from `@dnd-kit`'s `useSortable` and it already comes as a string (from console.logging: transition = "transform 200ms ease")